canonicalwebteam/discourse/parsers/category.py

```diff
@@ -0,0 +1,70 @@
+import re
+
+# Packages
+from slugify import slugify
+from bs4 import BeautifulSoup
+
+# Local
+from canonicalwebteam.discourse.parsers.base_parser import BaseParser
+
+
+class CategoryParser(BaseParser):
+    """
+    Parses a tables from a Discourse topic and stores them in a dictionary
+    """
+
+    def __init__(self, api, index_topic_id, url_prefix):
+        self.category_metadata = None
+        return super().__init__(api, index_topic_id, url_prefix)
+
+    def parse_index_topic(self):
+        """
+        Retrieve the index topic raw html content.
+        Find any data tables (distinguished by [deatils="NAME"]), store them
```

Comment:
I am not very acquainted with the raw form of data here so I would ask if this details parameter would always exist? Like is this something the maker of the topic need to take care of?

Reply:
It will not always exist, it is am optional flag you can add above a table mark it is as metadata. I choose `[details="NAME"] as this is the standard we use across discourse